spec/requests/mutations/resend_confirmation_spec.rb

```diff
@@ -0,0 +1,83 @@
+require 'rails_helper'
+
+RSpec.describe 'Resend confirmation' do
+  include_context 'with graphql query request'
+
+  let(:user)     { create(:user, confirmed_at: nil) }
+  let(:email)    { user.email }
+  let(:id)       { user.id }
+  let(:redirect) { Faker::Internet.url }
+  let(:query) do
+    <<-GRAPHQL
+      mutation {
+        userResendConfirmation(
+          email:"#{email}",
+          redirectUrl:"#{redirect}"
+        ) {
+          message
+          authenticable {
+            id
+            email
+          }
+        }
+      }
+    GRAPHQL
+  end
+
+  context 'when params are correct' do
+    it 'sends an email to the user with confirmation url and returns a success message' do
+      expect { post_request }.to(change(ActionMailer::Base.deliveries, :count).by(1))
```

Comment:
Please prefer [not_change](https://github.com/graphql-devise/graphql_devise/blob/master/spec/support/matchers/not_change_matcher.rb) matcher

Reply:
Ok, I can do that.

Reply:
Sorry, I got confused, this comment is actually ment to be posted on line 73 of this file